src/main/java/org/example/servletsHomework/dao/HashmapTextDao.java

```diff
@@ -0,0 +1,43 @@
+package org.example.servletsHomework.dao;
+
+import org.example.servletsHomework.model.Texts;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class HashmapTextDao implements TextDao {
+
+    private static final Map<Long, List<Texts>> map = new HashMap<>();
+
+    public boolean add(Long userId, Texts text) {
+        List<Texts> textsList = map.computeIfAbsent(userId, (key) -> new ArrayList<>());
+        return textsList.add(text);
+    }
+
+    public Texts getTextById(Long userId, Long textId) {
+        List<Texts> texts = map.computeIfAbsent(userId, (key) -> new ArrayList<>());
```

Comment:
you don't need create texts if users is not exists or user didn't create any text before